promql/value.go

```diff
@@ -194,6 +197,9 @@ type Sample struct {
 	H *histogram.FloatHistogram
 
 	Metric labels.Labels
+	// DropName is used to indicate whether the __name__ label should be dropped
+	// as part of the query evaluation.
+	DropName bool
```

Comment:
note: I did not add `json:"-"`  here as none of these fields seem be used for marshaling/unmarshaling (but happy to add it as a defensive measure if needed)

Reply:
Without that tag, it might just show up in JSON as a surprise. Let's better add the tag to not include this in marshaling.

Reply:
Sorry, now I get the actual meaning of your comment. The whole `Sample` type is never JSON-marshaled!

Then it makes sense what you say. We can just not have a json tag here. (I was conflating this with the `Series` type.)

I'll remove the tag directly, no need to send you through another roundtrip.